apps/web/src/modules/CourseSearch/components/FilterSection/constants.tsx

```diff
@@ -26,6 +26,17 @@ export const createGenEdCheckboxes: CreateCheckbox<GenEdChipKey>[] = [
   },
 ]
 
+export const createGradingCheckboxes: CreateCheckbox<GradingChipKey>[] = [
+  {
+    label: 'S/U Grade',
```

Comment:
Use `S/U` to be consistent with course detail page
```suggestion
    label: 'S/U',
```